### DISCUSSION TITLE
how to add a time system in flare ?

### DISCUSSION THREAD

**Discussion by BuzzLEquerre** [flareteam/flare-game#961] [General]
I am developing a mod for flare engine, and I would like to add a time system which allows you to have day and night when you walk around the map, could someone tell me if this is possible with flare engine, it would help me a lot.
Have a nice dau

**Comment by dorkster** [comment]
While not an explicit day/night system, one trick that the Ghostlore dev used was to draw a gradient image as a parallax layer to simulate the lighting. Here's what they used:

![daynight](https://github.com/user-attachments/assets/04d39e46-763b-46bc-b9bd-4fc319bb803d)

Then in a file in `maps/parallax/`, this was used to "animate" it:

```
[layer]
image=images/parallax/daynight.png
speed=-1
fixed_speed=-0.001,0.001
map_layer=object
```

**Comment by BuzzLEquerre** [comment]
so it’s possible! that's it, so I'll have to configure the animation and I could have time passing! Thank you very much, you saved me I was already imagining myself coding an entire system even though I have no coding knowledge xd
And is it possible to add a second parralax for example for snow ?

Thank you so much ! ;-)

**Reply by dorkster** [reply]
Yes, you can have multiple parallax layers on a map. For example, the star field and the fog in Flare's Hyperspace map are two separate layers.

**Reply by BuzzLEquerre** [reply]
How it's great !! Thanks you so mush :)

(I resume the development of my mod on Saturday I will share with you if I have other problems, here I continue my research during the holidays)
